lib/viewing/HasChildViews.js

```diff
@@ -94,17 +94,22 @@ var HasChildViews = {
     },
 
     removeUnrenderedChildView: function(viewRelationship) {
+        var unrenderedChildViews = this.unrenderedChildViews;
+
         if (!viewRelationship) {
             return;
         }
 
-        if (!this.unrenderedChildViews || !this.unrenderedChildViews.length) {
+        if (!unrenderedChildViews) {
```

Comment:
It wouldn't hurt to combine the two if/returns together, not sure if you noticed